src/main/java/com/wise23/chariteed/model/PatientGenerator.java

```diff
@@ -0,0 +1,34 @@
+package com.wise23.chariteed.model;
+
+import java.io.File;
+
+public class PatientGenerator {
+
+    private long id;
+    private String password;
+    private File file;
+
+    public long getId() {
```

Comment:
Do we need this?

Reply:
@JannesVolkens this would make development a bit easier. For example lombok has a built-in Builder pattern implementation which can be added through @Builder
Optional, but nice to use,
just add this to pom file


<dependency>
        <groupId>org.projectlombok</groupId>
        <artifactId>lombok</artifactId>
        <version>1.18.20</version>
        <scope>provided</scope>
    </dependency>

Reply:
`<dependency>
        <groupId>org.projectlombok</groupId>
        <artifactId>lombok</artifactId>
        <version>1.18.20</version>
        <scope>provided</scope>
    </dependency>`

Reply:
Done